build.sbt

```diff
@@ -1,12 +1,13 @@
 import ReleaseTransformations._
+import sbtcrossproject.CrossPlugin.autoImport.crossProject
 
 lazy val impSettings = Seq(
   organization := "org.spire-math",
   scalaVersion := "2.11.8",
```

Comment:
this also should be 2.11.12?

also, latest 2.12 scala is 2.12.8

Reply:
Correct on both counts, thanks!
I've changed the way `scalaVersion` is initialised to prevent this kind of error in the future - not a huge fan of giving meaning to the order of scala versions in `crossScalaVersions`, but at least it means we can never have an inconsistent state.